app/services/aws-helper.js

```diff
@@ -19,7 +19,7 @@ class AWSHelper {
     if (endpoint) {
       this.s3config.endpoint = endpoint
       this.s3config.s3BucketEndpoint = true
-      this.s3config.s3ForcePathStyle = true
+      this.s3config.forcePathStyle = true
```

Comment:
I think looking at https://docs.aws.amazon.com/AWSJavaScriptSDK/v3/latest/clients/client-s3/interfaces/s3clientconfig.html `s3BucketEndpoint` isn't required at all?

Reply:
Yeah, looking at the upgrade notes it seems that it's been renamed to lose the `s3` too - and might not even be needed: 

> s3BucketEndpoint
> 
> v2: Whether the provided endpoint addresses an individual bucket (false if it addresses the root API endpoint).
> v3: Changed to bucketEndpoint. See more in v3 reference for bucketEndpoint. Note that when set to true, you specify the request endpoint in the Bucket request parameter, the original endpoint will be overwritten. Whereas in v2, the request endpoint in client constructor overwrites the Bucket request parameter.

Will see what happens locally if I change/remove it...

Reply:
Works without it locally so have removed